plugins/node/opentelemetry-instrumentation-mongodb/src/instrumentation.ts

```diff
@@ -394,31 +543,51 @@ export class MongoDBInstrumentation extends InstrumentationBase<
           port = addressSegments[1];
         }
       }
-      if (host?.length && port?.length) {
-        span.setAttributes({
-          [SemanticAttributes.NET_HOST_NAME]: host,
-          [SemanticAttributes.NET_HOST_PORT]: port,
-        });
-      }
     }
 
     // The namespace is a combination of the database name and the name of the
     // collection or index, like so: [database-name].[collection-or-index-name].
     // It could be a string or an instance of MongoDBNamespace, as such we
     // always coerce to a string to extract db and collection.
     const [dbName, dbCollection] = ns.toString().split('.');
+    let commandObj: undefined | Record<string, unknown>;
+    // capture parameters within the query as well if enhancedDatabaseReporting is enabled.
+    if (command) {
+      commandObj = command.query ?? command.q ?? command;
+    }
```

Comment:
same here. a shortened writing of this code could look like this:
```js
const commandObj = command?.query ?? command?.q ?? command
```

Reply:
Fixed